### DISCUSSION TITLE
windows server 服务器系统如何搭建v2ray？

### DISCUSSION THREAD

**Discussion by FreeRolled** [v2fly/v2ray-core#1651] [General]
请问在 windows server 服务器系统如何搭建v2ray（服务器端）？网上查找一圈几乎都是Linux（centos，Debian，ubuntu）系统搭建的教程。

**Comment by ququ0755** [comment]
可以看看这个
https://www.zhuguodong.com/?id=383

**Reply by FreeRolled** [reply]
> 可以看看这个 https://www.zhuguodong.com/?id=383

服务器端那配置文件这么弄？Linux是一键命令生成的，Windows server 如何配置？

**Reply by xxx975** [reply]
> 可以看看这个 https://www.zhuguodong.com/?id=383

试了,不行.wv2ray.exe点不开, v2ray.exe点开后,卡在read config.json不动了

**Reply by haohao232323** [reply]
确实不行

**Reply by qq8087506** [reply]
试了那个教程不行，wv2ray.exe没有任何反应，v2ray.exe能打开但是也连不上。

**Reply by Julien-Chu** [reply]
经测试可行，
配置文件可以使用客户端生成，
由于是服务端，不弹CMD也属于正常现象，
只要有后台服务即可，也不需要点击其他exe

**Comment by snoopy33383622** [comment]
windows 可以搭建，我都搭建了几台的nginx+tls+ws了，跟LINUX一样的。手工配置脚本就行。我也是之前到处找资料没找到一个windows相关的。后来还是自己慢慢摸索出来的。效果还不错

**Reply by ziyelan** [reply]
能有空出个教程吗？我用的tcp协议很快就被封了ip，想学习一下怎么搭建

**Reply by 2279625821** [reply]
> windows 可以搭建，我都搭建了几台的nginx+tls+ws了，跟LINUX一样的。手工配置脚本就行。我也是之前到处找资料没找到一个windows相关的。后来还是自己慢慢摸索出来的。效果还不错

可以录个教程吗

**Reply by krebsgeorge** [reply]
用v2rayN生成一个不含tls的vmess-ws节点，然后另存位服务端配置。将配置文件传到windows server替换v2rayserver 配置文件。然后设置cloudflare  SSL 设置灵活  如果使用非80/443端口  在设置下origin rules,就可以实无nginx vmess-ws -tls节点

**Reply by Kukmoon** [reply]
我一开始试了 @KrebsGeorge 的方法，没成功。只要开启 WS 就没法连接。

然后，试了 @snoopy33383622  说的方法，搭建了一台 nginx+tls+ws+cdn 成功了，**关键是要在 Windows 上运行 nginx 和 v2ray core**（都是绿色软件，解压后直接运行），然后手工配置脚本的步骤与 Linux 基本一样，不同之处是脚本的存储目录不同。

**Comment by snoopy33383622** [comment]
很简单啊，比linux简单多了，你看你要装什么翻墙工具，一般有vmess 和 trojango，我搭建trojango比较多，直接下trojango的windows应用程序，然后配置文件就行,把trojango. exe设置为系统自动启动就行了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年1月27日(周五) 晚上9:22
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [v2fly/v2ray-core] windows server 服务器系统如何搭建v2ray？ (Discussion #1651)




  
windows 可以搭建，我都搭建了几台的nginx+tls+ws了，跟LINUX一样的。手工配置脚本就行。我也是之前到处找资料没找到一个windows相关的。后来还是自己慢慢摸索出来的。效果还不错
  
可以录个教程吗

**Comment by Mu-Chen2** [comment]
想问问各位大佬有成功在windows上搭建的了吗

**Comment by snoopy33383622** [comment]
很简单的，比linux简单多了，你下程序就下x64的，配置文件跟linux的一样，参考别人的教程就行。找个服务工具注册成服务开机自动运行就行

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月5日(周日) 凌晨0:15
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [v2fly/v2ray-core] windows server 服务器系统如何搭建v2ray？ (Discussion #1651)




 
想问问各位大佬有成功在windows上搭建的了吗

**Reply by Mu-Chen2** [reply]
感觉Linux有傻挂脚本和面板会更简单一些 我想要搭建Ipv6的服务端 但是半天都连不通

**Reply by mydogshitgold** [reply]
@Mu-Chen2 

> I want to build an IPv6 server, but I can't connect for a long time

Use Wireguard (see [wgcf](https://github.com/ViRb3/wgcf)) to connect Cloudflare WARP over IPv6 (2606:4700:d0::a29f:c001) then route 0.0.0.0/0 through the default wg gateway. 

To connect to an IPv6 only host over IPv4 via SSH you can use a CDN to get a public IPv4 server name route and then connect via [websocat](https://github.com/vi/websocat) or v2ray's  dokodemo-door + websocket secure.

**Reply by mydogshitgold** [reply]
Most cloud providers also offer VNC, which you can use for initial setup.

**Comment by garryforreg** [comment]
安装docker通过docker容器来部署服务端,或者直接开虚拟机来部署

**Comment by huacao123** [comment]
在阿里云上的windows服务器搭建需要自签名证书，配置后即可。
https://startup.aliyun.com/info/1074608.html
https://www.cotpear.com/2023/04/%E6%80%8E%E6%A8%A3%E5%9C%A8-windows-%E4%BC%BA%E6%9C%8D%E5%99%A8%E4%B8%8A%E6%90%AD%E5%BB%BA-vmess-%E7%AF%80%E9%BB%9E/
